packages/workflows/src/handlers/product/attach-sales-channel-to-products.ts

```diff
@@ -47,3 +42,7 @@ export async function attachSalesChannelToProducts({
     )
   )
 }
+
+attachSalesChannelToProducts.aliases = {
+  products: "products",
```

Comment:
question: Are we missing the productsHandleSalesChannelMap?

Reply:
I only needed one alias for one property in the data, the other one does not need alias, i can still export an alias in case someone need it